From e3b3bf8989c70864bb163dc537d3f63327845e7b Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 30 Jul 2021 20:35:17 +0000
Subject: [PATCH] libstdc++: fix incomplete type of net::CanonicalCookie

std::unordered_map of CookieHashSet requires full definition of
net::CanonicalCookie. Also remove no longer needed definitions
and includes.
---
 components/browsing_data/content/cookie_helper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/browsing_data/content/cookie_helper.h b/components/browsing_data/content/cookie_helper.h
index a444d26..11fbc5c 100644
--- a/components/browsing_data/content/cookie_helper.h
+++ b/components/browsing_data/content/cookie_helper.h
@@ -14,6 +14,7 @@
 #include "base/macros.h"
 #include "base/memory/ref_counted.h"
 #include "components/browsing_data/content/canonical_cookie_hash.h"
+#include "net/cookies/canonical_cookie.h"
 
 class GURL;
 
-- 
2.31.1

